pkgs/tools/package-management/disnix/default.nix

```diff
@@ -1,10 +1,11 @@
 { lib, stdenv, fetchurl, pkg-config, glib, libxml2, libxslt, getopt, gettext, dysnomia, libintl, libiconv }:
 
-stdenv.mkDerivation {
-  name = "disnix-0.10.1";
+stdenv.mkDerivation rec {
+  pname = "disnix";
+  version = "0.10.1";
 
   src = fetchurl {
-    url = "https://github.com/svanderburg/disnix/releases/download/disnix-0.10.1/disnix-0.10.1.tar.gz";
+    url = "https://github.com/svanderburg/disnix/releases/download/disnix-${version}/disnix-${version}.tar.gz";
```

Comment:
aybe you can also use ${pname} in the URL?

Reply:
We shouldn't substitute pname everywhere. This makes pname normalization harder than necessary and is also coincidence.